package.json

```diff
@@ -30,5 +30,16 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.17.0",
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "node-sass": "^4.13.0",
+    "react-redux": "^7.1.3",
+    "redux": "^4.0.5"
```

Comment:
Let's add this to the `dependencies` not `devDependencies`. Note that these packages will be used in the app and in production not only in development.


![image](https://user-images.githubusercontent.com/36057474/71618493-15579d00-2bc0-11ea-84bc-3849bd19bb03.png)